src/test/resources/MockData/compositePurchaseOrder.json

```diff
@@ -54,6 +55,7 @@
       "materialType" : "be44c321-ab73-43c4-a114-70f82fa13f17"
     },
     "fundDistribution" : [ {
+      "fundId" : "f5f1c495-7f7a-477a-b2e9-10eb043e37f4",
       "code" : "f5f1c495-7f7a-477a-b2e9-10eb043e37f4",
```

Comment:
Not really important but fundDistribution.code should be a short, human readable alphabetical code, e.g. USHIST

Reply:
Updated